keras/backend/mxnet_backend.py

```diff
@@ -199,18 +215,39 @@ def get_type(self):
         t = out_type[0]
         return _typename(t)
 
+    def __abs__(self):
+        return KerasSymbol(mx.sym.abs(self.symbol))
+
     def __add__(self, other):
-        return KerasSymbol(
-            mx.sym.broadcast_add(
-                lhs=self.symbol,
-                rhs=other.symbol))
+        if isinstance(other, KerasSymbol) or \
+            isinstance(other, KerasTensor):
```

Comment:
Keras do this:
x = K.variable(10)
y = K.placeholder() + x

placeholder returns a KerasSymbol while K.variable returns a tensor.